index.native.js

```diff
@@ -7,15 +7,44 @@ import Thunk from 'redux-thunk';
 
 import Config from './config';
 
-import reducer from './features/reducers';
+import reducers from './features/reducers';
 // FIXME Don't import private styles. Move common/shared styles to a feature in
 // base.
 import styles from './features/welcome/components/native/styles/Styles';
 
+import * as Actions from './features/actions';
 import { WelcomePage } from './features/welcome';
+import { Conference } from './features/conference';
```

Comment:
The same as my other comment on alphabetical order:

./features/actions
./features/conference
./features/welcome